src/pages/workspaces/workspace/jobHistory/CallTable.js

```diff
@@ -369,28 +360,17 @@ const CallTable = ({
                           TooltipTrigger,
                           {
                             content:
-                              'Approximate cost is calculated based on the list price of the VM, and does not include disk cost or any cloud account discounts.',
+                              "Approximate cost is calculated based on the list price of the VM, and does not include disk cost or any cloud account discounts. Tasks that successfully call cached don't incur cost.",
```

Comment:
Marketing: call caching doesn't "succeed" or "fail"; tasks either match cache or not

```suggestion
                              "Approximate cost is calculated based on the list price of the VM, and does not include disk cost or any cloud account discounts. Tasks that call cache don't incur cost.",
```